evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -131,8 +131,10 @@ def test_fit_predict_ts_with_X_and_y_index_out_of_sample(ts_data_seasonal):
     fh_ = forecasting.ForecastingHorizon(y[250:].index, is_relative=False)
 
     a_clf = sktime_arima.AutoARIMA()
-    clf = a_clf.fit(X=X[:250], y=y[:250])
-    y_pred_sk = clf.predict(fh=fh_, X=X[250:])
+    clf = a_clf.fit(y=y[:250])
```

Comment:
And to address your question, I agree entirely. The more time series datasets we can add with seasonality/various trends/etc, the better AutoARIMA can perform. Currently it can hold its own against some of the other regressor models but in the future it should most definitely outperform for most TS datasets.